appveyor.yml

```diff
@@ -1,18 +1,18 @@
 environment:
   matrix:
-  - JULIA_URL: "https://julialang-s3.julialang.org/bin/winnt/x86/0.6/julia-0.6-latest-win32.exe"
-    JAVA_HOME: C:\Program Files (x86)\Java\jdk1.7.0\
-  - JULIA_URL: "https://julialang-s3.julialang.org/bin/winnt/x64/0.6/julia-0.6-latest-win64.exe"
-    JAVA_HOME: C:\Program Files\Java\jdk1.7.0\
-  - JULIA_URL: "https://julialangnightlies-s3.julialang.org/bin/winnt/x86/julia-latest-win32.exe"
-    JAVA_HOME: C:\Program Files (x86)\Java\jdk1.7.0\
-  - JULIA_URL: "https://julialangnightlies-s3.julialang.org/bin/winnt/x64/julia-latest-win64.exe"
-    JAVA_HOME: C:\Program Files\Java\jdk1.7.0\
+  - julia_version: 0.7
+  - julia_version: 1.0
+  - julia_version: nightlies
```

Comment:
I was mistaking here, it should be latest, not nightlies, for the Appveyor script (it's silly that it's different in the two scripts, .travis.yml and Appveyor.yml)